### PR TITLE
Update cocli docs with CoTS extension + fixes

### PR DESCRIPTION
**Pull request by SabreenKaur**
This change updates the documentation to include instructions on using the cocli cli tools with CoTS (Concise Ta Store). Additionally there are some fixes to the cocli cots display tool.

The new diagram is rendered at https://github.com/veraison/corim/tree/cocli-cots-docs-fixes/cocli#visual-synopsis-of-the-available-commands

This addresses #80 #74

### CODE REVIEW

**Review by thomas-fossati — APPROVED** [review]

**Comment by yogeshbdeshpande** [comment]
@SabreenKaur As we are extending the documentation to absorb Cots introduction. Please enhance 
CoRIM README.md to extend the sentence below:
_to provide a user friendly command line interface for working with CoRIM, CoMID and CoSWID and CoTS._

File: cocli/README.md [hunk 1]
```diff
@@ -156,21 +156,128 @@ $ cocli comid display -f m1.cbor \
                     -d yet-another-comid-folder/
 ```
 
+## CoTSs manipulation
+
+The `cots` subcommand allows you to create, display and validate CoTSs.
+
+### Create
+
+Use the `cots create` subcommand to create a CBOR-encoded CoTS:
```
Comment (yogeshbdeshpande) [inline cocli/README.md:165]: Please refer clearly what are the supplied inputs, like as done in CoMID above. From the below description it is not clear is c1 a template and what is tas_dir?

File: cocli/README.md [hunk 2]
```diff
@@ -156,21 +156,128 @@ $ cocli comid display -f m1.cbor \
...
+
+The CBOR-encoded CoTS file is stored in the current working directory with a
+name derived from its environment template.  If you want, you can specify a different
+target directory and file name using the `--output` command line switch (abbrev. `-o`)
+```
+$ cocli cots create --environment c1.json --tas tas_dir --output /tmp/myCots.cbor
+>> created "/tmp/myCots.cbor"
+```
+Note that the output directory, as well as all its parent directories, MUST
+pre-exist.
```
Comment (yogeshbdeshpande) [inline cocli/README.md:182]:
```suggestion
Note that the output directory, as well as all its parent directories, MUST pre-exist.
```

**Review by yogeshbdeshpande — COMMENTED** [review]
LGTM, in general: left few minor comments to look for..

**Comment by SabreenKaur** [comment]
> @SabreenKaur : nice diagram: We should extend this (may be in another PR) to absorb the Issue #74 .

Sure, I will extend the diagram to have the submit command in this PR.

**Pull request merged**